dist/constants.json

```diff
@@ -5883,931 +5883,931 @@
-        "label": "DENVER VA CLINIC"
+        "label": "York Street VA Clinic"
       },
       {
         "value": "442QE",
-        "label": "STERLING VA CLINIC"
+        "label": "Sterling VA Clinic"
       },
       {
         "value": "575QC",
-        "label": "GRAND JUNCTION VA MOBILE CLIN"
+        "label": "Grand Junction VA Mobile Clinic"
       },
       {
         "value": "554",
-        "label": "ROCKY MOUNTAIN REGIONAL VAMC"
+        "label": "Rocky Mountain Regional VA Medical Center"
       },
       {
         "value": "575",
-        "label": "GRAND JUNCTION VAMC"
+        "label": "Grand Junction VA Medical Center Vocational Rehabilitation Counselor"
       }
     ],
     "HI": [
       {
         "value": "459GA",
-        "label": "MAUI VA CLINIC"
+        "label": "Maui VA Clinic"
       },
       {
         "value": "459GB",
-        "label": "HILO CBOC"
+        "label": "Hilo VA Clinic"
       },
       {
         "value": "459GC",
-        "label": "KONA CBOC"
+        "label": "Kailua-Kona VA Clinic"
       },
       {
         "value": "459GD",
-        "label": "KAUAI CBOC"
+        "label": "Lihue VA Clinic"
       },
       {
         "value": "459GG",
-        "label": "LEEWARD OAHU CBOC"
+        "label": "Leeward Oahu VA Clinic"
       },
       {
         "value": "459",
-        "label": "SPARK M. MATSUNAGA VAMC"
+        "label": "Honolulu Regional Benefit Office"
       }
     ],
     "GU": [
       {
         "value": "459GE",
-        "label": "GUAM CBOC"
+        "label": "Guam VA Clinic"
       }
     ],
     "ME": [
       {
         "value": "402GA",
-        "label": "CARIBOU MAINE CBOC"
+        "label": "Caribou VA Clinic"
       },
       {
         "value": "402GB",
-        "label": "CALAIS MAINE CBOC"
+        "label": "Calais VA Clinic"
       },
       {
         "value": "402GC",
-        "label": "RUMFORD MAINE CBOC"
+        "label": "Rumford VA Clinic"
       },
       {
         "value": "402GD",
-        "label": "SACO MAINE CBOC"
+        "label": "Saco VA Clinic"
       },
       {
         "value": "402HB",
-        "label": "BANGOR MAINE CBOC"
+        "label": "Bangor VA Clinic"
       },
       {
         "value": "402HC",
-        "label": "PORTLAND MAINE MH ORC"
+        "label": "Portland VA Clinic"
       },
       {
         "value": "402HL",
-        "label": "RURAL AND COMNTY HLTCARE MOC"
+        "label": "Bingham VA Mobile Clinic"
       },
       {
         "value": "402GE",
-        "label": "LEWISTON AUBURN CBOC"
+        "label": "Lewiston VA Clinic"
       },
       {
         "value": "402GF",
-        "label": "LINCOLN VA CLINIC"
+        "label": "Lincoln VA Clinic"
       },
       {
         "value": "402",
-        "label": "MAINE VA HCS"
+        "label": "Togus Regional Benefit Office"
       }
     ],
     "NH": [
       {
         "value": "405HC",
-        "label": "ST. JOHNSBURY VERMONT OUTREACH CLINIC"
+        "label": "Littleton VA Clinic"
```

Comment:
I don't see any reference to **ST. JOHNSBURY VERMONT OUTREACH CLINIC** in NH. So it may be safe to change this one.

https://www.va.gov/find-locations/facility/vha_405HC

Reply:
I'd leave this one as is, since we can't confirm the name change. **ST. JOHNSBURY VERMONT OUTREACH CLINIC** sounds like it was in Vermont, but **Littleton VA Clinic** is in Littleton, NH. Maybe the code was rotated to another facility.